app/services/user_update.rb

```diff
@@ -24,6 +24,49 @@ def update
 
 private
 
+  class SupportedPermissionsSanitizer
```

Comment:
Oh, interesting, I didn't spot the other sanitiser.  I'm not sure sanitizer is even the right name for this thing (it has connotations of rails own strong parameters stuff which is key focussed, when in fact we're focussed on values in this thing).  I'll move it though.